features/help.feature

```diff
@@ -63,10 +63,15 @@ Feature: help command
     When running "exo help deploy" in the terminal
     Then I see:
       """
-      Deploys Exosphere application to the cloud
+      Deploys an Exosphere application to the cloud
 
       Usage:
-        exo deploy
+        exo deploy [command]
+
+      Available Commands:
+        apply       Deploys an Exosphere application to the cloud
+        build       Prepares an Exosphere application to be deployed
```

Comment:
That what I called it initially, but `build` also pushes your Docker images to ECR and generates your `.tf` files, so it would theoretically be run each time the user makes changes to their app